packages/belos/src/BelosBlockGmresSolMgr.hpp

```diff
@@ -578,27 +578,17 @@ void BlockGmresSolMgr<ScalarType,MV,OP>::setParameters( const Teuchos::RCP<Teuch
   // Check if the orthogonalization changed.
   if (params->isParameter("Orthogonalization")) {
     std::string tempOrthoType = params->get("Orthogonalization",orthoType_default_);
-#ifdef HAVE_BELOS_TSQR
-    TEUCHOS_TEST_FOR_EXCEPTION( tempOrthoType != "DGKS" && tempOrthoType != "ICGS" && tempOrthoType != "IMGS" && tempOrthoType != "TSQR",
-                        std::invalid_argument,
-                        "Belos::BlockGmresSolMgr: \"Orthogonalization\" must be either \"DGKS\", \"ICGS\", or \"IMGS\", or \"TSQR\".");
-#else
-    TEUCHOS_TEST_FOR_EXCEPTION( tempOrthoType != "DGKS" && tempOrthoType != "ICGS" && tempOrthoType != "IMGS",
-                        std::invalid_argument,
-                        "Belos::BlockGmresSolMgr: \"Orthogonalization\" must be either \"DGKS\", \"ICGS\", or \"IMGS\".");
-#endif
     if (tempOrthoType != orthoType_) {
       orthoType_ = tempOrthoType;
       params_->set("Orthogonalization", orthoType_);
       // Create orthogonalization manager
       Belos::OrthoManagerFactory<ScalarType, MV, OP> factory;
-      Teuchos::RCP<Belos::OutputManager<ScalarType>> outputOrtho;   // can be null
       Teuchos::RCP<Teuchos::ParameterList> paramsOrtho;   // can be null
       if (orthoType_=="DGKS" && orthoKappa_ > 0) {
         paramsOrtho->set ("depTol", orthoKappa_ );
       }
 
-      ortho_ = factory.makeMatOrthoManager (orthoType_, Teuchos::null, outputOrtho, "Belos", paramsOrtho);
```

Comment:
Fun fact: in `makeMatOrthoManager`, the printer doesn't do anything ;-) .  This change is harmless, though.  Also, if `makeMatOrthoManager` ever does actually use the printer, then you'll need to add a method to tell the MatOrthoManager about any changes to the printer.

Reply:
Okay.  I was imagining it might get passed to the orthogonalization manager somewhere...  Where is orthogonalization output managed?

Reply:
@jennloe Perhaps @hkthorn can explain that better.